src/java.base/macosx/native/libjli/java_md_macosx.m

```diff
@@ -429,18 +429,6 @@ static void MacOSXStartup(int argc, char *argv[]) {
                 return JNI_TRUE;
             }
         }
-        /* ensure storage for path + /jre + NULL */
```

Comment:
The corresponding changes in JDK-8329862 included adding a call to JLI_TraceLauncher.
Such as:
`    JLI_TraceLauncher("Attempt to get JDK installation root from launcher executable path\n");
`
Adding a similar trace message would be consistent across platforms.

Reply:
That's a good point. I've now updated the PR to add the same log message as the one present in unix/windows variant of this code.